### PR TITLE
Catch invalid json.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/134/
Test PASSed.

File: theblues/plans.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def get_plans(self, reference):
                 price=plan.get('price')), json))
         except (KeyError, TypeError, ValueError) as err:
```
Comment (bac) [inline theblues/plans.py:46]: I think we should have another bare except clause to return a generic ServerError if some unexpected exception is raised.  Here and below.

**Comment by bac** [comment]
:+1: with suggestion

**Comment by kadams54** [comment]
:+1: from me as well. Thanks!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/136/
Test PASSed.

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**